README.md

```diff
@@ -54,6 +54,7 @@ A simple plugin for developing AutoHotkey scripts. The following features are av
 
 - Syntax highlighting (under construction; no error checking available currently)
 - Run configurations
+- Quick documentation
 - More to come in the future...
 
```

Comment:
Thanks for updating the readme. Please also add a short blurb in the plugin description in this seciton so that people viewing the plugin within the IntelliJ plugins marketplace Settings menu will be aware about the dependency (unfortunately the majority of plugin users will never view the README lol).

Something simple like this should work fine:
```
...
- More to come in the future...

Dependencies:

- Archive Browser plugin (for documentation assistance)
...
```

Reply:
oh ok. I guess I delete the standalone section I added below then